expect/mod.ts

```diff
@@ -48,25 +48,29 @@
  * - `toThrowErrorMatchingSnapShot`
  * - `toThrowErrorMatchingInlineSnapShot`
  *
- * The following asymmetric matchers are not supported yet:
- * - `expect.anything`
- * - `expect.any`
- * - `expect.arrayContaining`
- * - `expect.not.arrayContaining`
- * - `expect.closedTo`
- * - `expect.objectContaining`
- * - `expect.not.objectContaining`
- * - `expect.stringContaining`
- * - `expect.not.stringContaining`
- * - `expect.stringMatching`
- * - `expect.not.stringMatching`
+ * The asymmetric matchers are not yet fully supported:
+ * - Supported:
+ *   - `expect.anything`
+ *   - `expect.any`
+ *   - `expect.arrayContaining`
+ *   - `expect.not.arrayContaining`
+ *   - `expect.closedTo`
+ *   - `expect.stringContaining`
+ *   - `expect.not.stringContaining`
+ *   - `expect.stringMatching`
+ *   - `expect.not.stringMatching`
+ * - Not supported:
+ *   - `expect.objectContaining`
+ *   - `expect.not.objectContaining`
  *
- * The following uitlities are not supported yet:
- * - `expect.assertions`
- * - `expect.hasAssertions`
- * - `expect.addEqualityTester`
- * - `expect.addSnapshotSerializer`
- * - `expect.extend`
+ * The uitlities are not yet fully supported:
+ * - Supported:
+ *   - `expect.addEqualityTester`
+ *   - `expect.extend`
+ * - Not supported:
+ *   - `expect.assertions`
+ *   - `expect.hasAssertions`
+ *   - `expect.addSnapshotSerializer`
```

Comment:
Ditto. This doesn't make sense for the same reason as the above list. Please correct and clarify.